tally_ho/apps/tally/tests/views/test_corrections.py

```diff
@@ -270,10 +269,9 @@ def test_corrections_general_post_corrections(self):
         self.assertEqual(
             Result.objects.filter(result_form=result_form).count(), 2)
         session = {'result_form': result_form.pk}
-        de_1_suffix = getattr(settings, "DE_1_SUFFIX")
         post_data = {
-            'candidate_general_%s%s' % (result_form.results.all()[
-                0].candidate.pk, de_1_suffix): 2,
+            'candidate_general_%s' % result_form.results.all()[
+                0].candidate.pk: 2,
```

Comment:
Why are we electing not to use [f-strings](https://realpython.com/python-f-strings/) here?  (Nitpicky, I know, but %-formatting is not recommended since Python 3.6)

Reply:
This has been repeated several places in the code base, i think it's best i create a new issue, which will refactor code that was written in the python 2.7 way to 3.6.

Reply:
For now i'll fix this though.

Reply:
> This has been repeated several places in the code base, i think it's best i create a new issue, which will refactor code that was written in the python 2.7 way to 3.6.

An alternative may be to update every file that we touch going forward, until everything is fixed.

Reply:
Done